tests/testthat/test-pcens.R

```diff
@@ -1,21 +1,20 @@
 test_that("new_pcens creates object with correct structure", {
-  pdist_name <- "pgamma"
   pdist <- pgamma
-  dprimary_name <- "dunif"
   dprimary <- dunif
   shape <- 2
   rate <- 1
 
   obj <- new_pcens(
     pdist,
     dprimary, list(),
-    pdist_name, dprimary_name,
     shape = shape, rate = rate
   )
 
   expect_s3_class(obj, "pcens_pgamma_dunif")
-  expect_identical(obj$pdist, pgamma)
-  expect_identical(obj$dprimary, dunif)
+  expect_identical(body(obj$pdist), body(pgamma))
+  expect_identical(formals(obj$pdist), formals(pgamma))
+  expect_identical(body(obj$dprimary), body(dunif))
+  expect_identical(formals(obj$dprimary), formals(dunif))
```

Comment:
n.b. expect_identical tests attributes, which no longer match, hence need to change this test.